src/classes/HiFiAudioAPIData.ts

```diff
@@ -44,6 +44,12 @@ export class HiFiAudioAPIData {
      * A volume level below this value is considered background noise and will be smoothly gated off.
      * The floating point value is specified in dBFS (decibels relative to full scale) with values between -96 dB (indicating no gating)
      * and 0 dB (effectively muting the input from this user). It is in the same decibel units as the VolumeDecibels component of UserDataSubscription.
+     * Setting this value to `NaN` or `null` will cause the volume threshold from the space to be used instead.
+     * 
+     * **COMPATIBILITY WARNING:** In the future, the High Fidelity Audio API server will only fall back to the space volume threshold
+     * if the threshold is `NaN`.
```

Comment:
I didn't follow what this sentence means. It seems to imply that currently (i.e., not yet the future behavior), there is some way to fall back to the space volume threshold other than setting the value to NaN, and that in the future, .... ?

Reply:
Does the sentence above the compatibility warning help clarify, or is it still confusing?

Is it just this particular compatibility warning that is confusing, or are the others confusing as well?

Reply:
I'm not sure. 
- Part of my confusion is whether it is related to the sentence above. On that issue, maybe it would be clearer to state just the going-forward behavior detached above, and then in the Compatibility Warning note, to specify the additional current behavior along with how that will change. (Maybe also including what the future interpretation will be for null, or zero in the other cases below.)
- Part of my confusion may be about the behavior of 0 vs null here and below.
- And part of my confusion is about what happens if not specified at all. The use of "only fall back to the space volume threshold if the threshold is 'NaN'" seems to say that it has to be explicitly set, but then what would meaningfully happen if not set?

Reply:
I have updated the compatibility warnings and preceding text and added some details.